crates/router/src/core/unified_connector_service.rs

```diff
@@ -909,3 +938,70 @@ where
 
     router_result
 }
+
+#[derive(serde::Serialize)]
+pub struct ComparisonData {
+    pub hyperswitch_data: serde_json::Value,
+    pub unified_connector_service_data: serde_json::Value,
```

Comment:
Nit: We may have to use `Secret<Value>` depending on whether we're using masked serialization or not.